docker/compose.yaml

```diff
@@ -16,16 +16,57 @@ x-environment: &environment
     __SUPERBLOCKS_AGENT_SERVER_URL: ${__SUPERBLOCKS_AGENT_SERVER_URL:-https://app.superblocks.com}
 
 services:
+  proxy:
+    # Configurable variables for traefik to set up https
+    # SUPERBLOCKS_PROXY_ADMIN_EMAIL
+    # SUPERBLOCKS_PROXY_REPLICA_COUNT, default 0
+    # SUPERBLOCKS_LETSENCRYPT_DIR, default letsencrypt
+    image: traefik:v2.8
+    deploy:
+      mode: replicated
```

Comment:
Not certain if this is necessary. Let's remove if it isn't

Reply:
Removed